tests/foreman/ui/test_computeresource.py

```diff
@@ -51,6 +51,9 @@ def rhev_data():
         'storage_domain': settings.rhev.storage_domain,
         'cert': settings.rhev.ca_cert,
     }
+    if 'INTERFACE' in settings.rhev:
```

Comment:
`hasattr` check would be slightly better here - using this syntax is getting a list of keys and checking membership, and will be a bit sensitive to capitalization in the config.